config/dev.exs

```diff
@@ -62,11 +62,10 @@ config :code_corps, CodeCorps.Mailer,
 config :code_corps,
   postmark_receipt_template: "123"
 
-# If the dev environment has no CLOUDEX_API_KEY set, we want the app to still run,
-# with cloudex in test API mode
+# If the dev environment has no CLOUDEX_API_KEY set, we want the app
+# to still run, with cloudex in test API mode
 if System.get_env("CLOUDEX_API_KEY") == nil do
-  config :cloudex, :cloudinary_api, Cloudex.CloudinaryApi.Test
-  config :cloudex, api_key: "test_key", secret: "test_secret", cloud_name: "test_cloud_name"
-
   IO.puts("NOTE: No Cloudex configuration found. Cloudex is runnning in test mode.")
+  config :code_corps, :cloudex, CloudexTest
```

Comment:
Yeah, I'm going to move this out of `helpers` too and into a dedicated `cloudex`.